### PR TITLE
bump kafka version to 1.1.1

### PR DESCRIPTION
**Pull request by eskabetxe**
upgrade kafka verstion to most recent version posible

### CODE REVIEW

**Comment by eskabetxe** [comment]
@kokosing clould you check

File: pom.xml [hunk 1]
```diff
@@ -948,7 +948,7 @@
             <dependency>
                 <groupId>org.xerial.snappy</groupId>
                 <artifactId>snappy-java</artifactId>
-                <version>1.1.2.6</version>
```
Comment (kokosing) [inline pom.xml]: this is going to change a snappy version for entire project (not only for kafka). So IMO it should go in separate commit (if possible).

File: presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/EmbeddedKafka.java [hunk 2]
```diff
@@ -91,7 +92,7 @@ public static EmbeddedKafka createEmbeddedKafka(Properties overrideProperties)
                 .build();
 
         KafkaConfig config = new KafkaConfig(properties);
-        this.kafkaServer = new KafkaServer(config, Time.SYSTEM, Option.empty(), List.make(0, null));
+        this.kafkaServer = new KafkaServer(config, Time.SYSTEM, Option.empty(), KafkaMetricsReporter$.MODULE$.startReporters(new VerifiableProperties(new Properties())));
```
Comment (kokosing) [inline presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/EmbeddedKafka.java]: What is this?

Comment (eskabetxe) [reply presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/EmbeddedKafka.java]: the new version needs a KafkaMetricReporter..
changed to use java version..

File: pom.xml (hunk 1 above)
Comment (eskabetxe) [reply pom.xml]: its necessary by kafak update..
created a separated commit

**Comment by eskabetxe** [comment]
@cla-bot check

**Comment by kokosing** [comment]
Merged, thanks!
